src/directives/profile.js

```diff
@@ -5,7 +5,7 @@
  * Example:
  *
  * <div ngeo-profile="ctrl.profileData"
- *      ngeo-profile-ooptions="ctrl.profileOptions"></div>
+ *      ngeo-profile-options="ctrl.profileOptions"></div>
  *
  * Note: "ctrl.profileOptions" is of type ngeox.profile.ProfileOptions.
```

Comment:
The option is documented in the ngeox extern. Do I add a link to the typedef like this: {@link ngeox.profile.ProfileOptions}?